spec/mocks/game_start.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'game'
+require_relative 'prompter'
+require_relative 'view'
+
+puts "Hi"
+
+# view = View.new
```

Comment:
Should this be commented out? You only want Pull Requests to contain a working app with nothing but production code.